docs/modules/ROOT/pages/how-to-guides/proc_creating_your_own_environment.adoc

```diff
@@ -0,0 +1,33 @@
+= Creating your own environment 
+
+Use {ProductName} to continuously build and test your application. Use your own cluster as your deployment environment. By using your own cluster outside Red Hat, you have more control over where your application is deployed, and you can ensure it meets your specific needs. If you bring your own cluster you can still access all the features and benefits of {ProductName}. 
+
+.Prerequisites
+
+* You must have access to credentials with full read and write access to the namespace that you provided to {ProductName}.   
+* When you upload a kubeconfig file, it must include a credential with full read and write access to your provided namespace. 
```

Comment:
snap, @jc-berger, i forgot that we have to put file names in monocode (with the backticks `code`). i won't hold this up for something so small, but come back soon and add? (i need to do the same in mine, i just realized!)